src/m365/planner/commands/task/task-checklistitem-list.ts

```diff
@@ -0,0 +1,62 @@
+import { Logger } from "../../../../cli";
+import { CommandOption } from "../../../../Command";
+import { accessToken } from "../../../../utils";
+import GlobalOptions from "../../../../GlobalOptions";
+import Auth from "../../../../Auth";
+import request from "../../../../request";
+import GraphCommand from "../../../base/GraphCommand";
+import commands from "../../commands";
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  taskId: string;
+}
+
+class PlannerTaskChecklistitemListCommand extends GraphCommand {
```

Comment:
Could you please ensure that we have a public defaultProperties(): string[] | undefined  implemented?

Reply:
@plamber @Adam-it - Thank you for your review, Initially I tried to implement defaultProperties but have no clue on how to use it when fetching child elements. Currently checklists is a child elements of TaskDetails endpoint.  I checked similar [ references list command](https://github.com/pnp/cli-microsoft365/blob/main/src/m365/planner/commands/task/task-reference-list.ts) but defaultProperties is not implemented in that command.  I tried different variations like ["title"] ["checklists.title"] but everytime it display as empty. Any help/direction to implement to show child elements properties will be great appreciated. Here is the sample output of command where checklist is child element.
![image](https://user-images.githubusercontent.com/14875265/169834228-b368beac-a9ef-48c6-bfc9-1944b677b19f.png)

Reply:
@plamber I am not sure if providing defaultProperties is like possible in this case as id of task are property names in the structure so we would need to provide something like `return ['43039'];`  🤔

Reply:
@plamber  - I have rebase the branch. I am not sure how we will be able to use defaultProperties in this case.

Reply:
Thank you for the feedback. Lets keep it like this

Reply:
Thank you!

Reply:
In this case, I'd suggest that in text mode, we replace the key-value structure, with an array of the values, so instead of:

```json
{
  "checklist": {
    "4123": {
      "title": "item 1"
    },
    "4234": {
      "tittle": "item 2"
    }
  }
}
```

we'd have:

```json
[
  {
    "title": "item 1"
  },
  {
    "title": "item 2"
  }
]
```

Because this is an array we could use defaultProperties just fine.

If we need the keys for something, like manipulation of the checklist items, we can add them to the objects in the array under the `key` property.

If we leave the current output as-is, then it will be unreadable in text output, so I'd say, let's revisit it before we process this PR.

Reply:
@waldekmastykarz - Thank you for your feedback. As I understand from the input, please let me know if this works?
Initial JSON(Original Ouput from graph api) - 
![image](https://user-images.githubusercontent.com/14875265/171202246-ebaa860d-051a-4344-b359-2a5a5d6110a4.png)
Modified JSON(converted to array) -
![image](https://user-images.githubusercontent.com/14875265/171202636-b0b009db-2d89-4dd1-9a5e-c7df251e5e48.png)
Using default properties- 
![image](https://user-images.githubusercontent.com/14875265/171202844-34057444-983b-4b39-aaae-db1f317da446.png)

Reply:
@waldekmastykarz - I got you. Thank you. I will update the code.

Reply:
Hi @waldekmastykarz - Modified the code, now the output looks like below in test and json format
![image](https://user-images.githubusercontent.com/14875265/171281225-678ce5c6-7f6b-4c23-a229-169099d721bf.png)

Reply:
Perfect! Thank you so much for such a quick turnaround 👏